bibisco/src/ApplicationResources_es.properties

```diff
@@ -120,12 +147,12 @@ jsp.menu.suggestedReadings=lecturas sugeridas
 
 #welcome
 jsp.welcome.h1=\u00A1Bienvenido!
-jsp.welcome.step1.p.1=Antes de comenzar, selecciona tu idioma:
-jsp.welcome.step2.p.1=Y ahora selecciona donde bibisco guardar\u00E1 tus proyectos:
+jsp.welcome.step1.p.1=Antes de comenzar, seleccione su idioma:
+jsp.welcome.step2.p.1=Y ahora seleccione donde bibisco guardar\u00E1 sus proyectos:

```

Comment:
Why "seleccione" instead of "selecciona"? bibisco talks to user in informal way.